### DISCUSSION TITLE
Table "Document" not found - on setup

### DISCUSSION THREAD

**Discussion by akim-green** [OrchardCMS/OrchardCore#8762] [Q&A]
Has anyone encountered this DB error on initial setup, happens with MySQL and SQLite:

`MySqlConnector.MySqlException: 'Table 'dev.document' doesn't exist' I get either that or Table identifiers already exist if I run it again. Not sure how to solve I've tried dropping the schema, deleting my App_Data, cleaning and rebuilding the solutions, updating to the latest packages and updating my VS, nothing seemed to work.

Any advice would be greatly appreciated.
`

**Comment by Skrypt** [comment]
My first guess is that you don't have enough privilege on the database table to create them. I'm intrigued though by the prefix name of your table which contains a dot. Or is it the table name?
